pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
I want it to be differentiated from a.group.GroupVersion, which the (comment)[https://github.com/kubernetes/kubernetes/pull/18379/files#diff-fdfac9813673596faeda07fdce9207e2R84] says is the external version. We can call it "ExternalOptionsVersion" or other better names, anything more specific than "ExternalVersion".